<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QDialog" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="estimate_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="property_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>500000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="downpayment_spinbox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="property_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Property value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rent_lineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rent_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Monthly rent</string>
   </property>
  </widget>
  <widget class="QLabel" name="downpayment_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Down Payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="interest_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amortization_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>25 years</string>
   </property>
   <property name="cursorPosition">
    <number>8</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amortization_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amortization</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_growth_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Property value growth rate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="property_growth_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>3.00%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interest_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>2.95%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="property_tax_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>1.5%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_tax_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annual property tax</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="insurance_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>1.5%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="insurance_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annual insurance</string>
   </property>
  </widget>
  <widget class="QLabel" name="maintenance_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annual maintenance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maintenance_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>1.00%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="rent_increase_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annual rent increase</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rent_increase_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>1.20%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="investment_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annual investment return</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="investment_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>8.00%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="closing_cost_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="specialValueText">
    <string>4.00%</string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_growth_label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Buying closing cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_years_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of years</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="years_lineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>5 years</string>
   </property>
   <property name="cursorPosition">
    <number>7</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
